.github/workflows/desul-version-check.yml

```
@@ -27,4 +27,4 @@ jobs:
         rm desul-atomics/atomics/CMakeLists.txt
         rm desul-atomics/atomics/desul_atomicsConfig.cmake.in
         rm desul-atomics/atomics/src/common.cpp
-        diff --brief --recursive tpls/desul desul-atomics/atomics
+        diff --unified 3 --recursive tpls/desul desul-atomics/atomics
```

There should be an `=` between unified and 3, otherwise 3 is parsed a filename to diff
```suggestion
        diff --unified=3 --recursive tpls/desul desul-atomics/atomics
```

---

Thanks for fixing. Can you apply it in my behalf. I am on my cell and the gh app won't let me apply both patches in a single commit.